src/providers/data_providers/mockBusinessImageProvider.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { Image, ImageProviderName } from '../../models/data_models/image';
+
+export class MockBusinessImageProvider extends BaseBusinessImageProvider {
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		return Promise.resolve([
+			{
+				source: 'https://s3-media0.fl.yelpcdn.com/bphoto/AZllw6P3nvB0BCmsw5kMjQ/l.jpg',
+				provider: ImageProviderName.mock,
+			},
+			{
+				source: 'https://www.ubburger.com/wp-content/uploads/2016/01/locations_img.jpg',
+				provider: ImageProviderName.mock,
+			},
+			{
+				source: 'https://live.staticflickr.com/7295/10218344623_a6764810d4_b.jpg',
```

Comment:
We might want to make to make our mock outputs consistent with each other. ie the mock extraction provider is returning two extracted images, should we make this mock data provider return those same two images?